bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -271,6 +284,28 @@
 		<state min="0" step="1" max="100" pattern="%.0f %unit%" readOnly="false"/>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 	</channel-type>
+	<channel-type id="limitTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Charging Limit Temperature</label>
+		<description>Until which Temperature should the heating device be charged</description>
+		<category>Temperature</category>
+		<state min="0" step="1" max="100" pattern="%.0f %unit%" readOnly="false"/>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="effectiveLimitSoC">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Effective Charging Limit SoC</label>
```

Comment:
```suggestion
		<label>Effective Charging Limit</label>
```

When changed, the poperties file also needs adjustment

Reply:
I wouldn't like to change this, as `effectiveLimitSoC` is the name used by evcc

Reply:
Labels should be short 2-3 words and < 25 characters. This label exceeds both, i don't think SoC adds any value to this label, it is also mentioned in the description.

Reply:
ok, changed ...